hdk/cl/developer_designs/cl_firesim/build/scripts/create_dcp_from_cl.tcl

```diff
@@ -156,10 +156,20 @@ set_msg_config -severity "CRITICAL WARNING" -string "WRAPPER_INST/SH" -suppress
 set_msg_config -severity "WARNING"          -string "WRAPPER_INST/SH" -suppress
 
 # Promote the following critical warnings to errors to prevent AGFI generation
+# A terrifying warning that may be symptomatic of unsound BRAM inference:
+#
+# > The Block RAM <RAM> get memory collision
+# > error if read and write address collide. Use attribute (* rw_addr_collision= "yes" *)
+# > to avoid collision
```

Comment:
```suggestion
# > to avoid collision
# See https://www.beyond-circuits.com/wordpress/2019/10/ram-address-conflicts-and-a-vivado-synthesis-bug/ for discussion.
```